gitrevise/merge.py

```diff
@@ -255,6 +267,9 @@ def merge_files(
             "--quiet",
             # Send results to stdout instead of overwriting "current file".
             "--stdout",
+            # Ensure markers are the expected length to ensure consistent rerere results.
+            "--marker-size",
+            str(marker_size),
```

Comment:
nice, I thought there was no way to override attributes.
I'd probably put this in a separate commit, apart from the refactoring.

Reply:
It's part of the "parameterize marker size" commit. I think that seems like the right place to me?

Note that this does not override attributes: they weren't and aren't used at all here. This is merging arbitrary temp files and `git merge-file` doesn't refer to the repo at all other than to check user preference for diff3 style, which isn't relevant here.

The note about using `git check-attr` to determine `conflict-marker-size` is just so that the `rerere` conflicts here would be cross-compatible with the ones created and used by vanilla `git rebase`/`git merge`.

Reply:
ah sorry I thought this commit already had a behavior change